src/lib/components/messages/inbox/inbox.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import {
+  LargeHeadline,
+} from "../typography"
+
+import {
+  ScrollView,
+} from "react-native"
+
+import ConversationSnippet from "./conversationsnippet"
+
+export class Inbox extends React.Component<any, any> {
+  renderConversations() {
+    return (this.props.conversations || []).map(conversation => (
+      <ConversationSnippet conversation={conversation} />
+    ))
+  }
+
+  render() {
+    return (
+      <ScrollView>
+          <LargeHeadline style={{marginTop: 10}}>Messages</LargeHeadline>
+          {this.renderConversations()}
+      </ScrollView>
+    )
+  }
+}
+
+export default Relay.createContainer(Inbox, {
+  fragments: {
+    conversations: () => Relay.QL`
+      fragment on ConversationType @relay(plural: true) {
```

Comment:
as we discussed, this conversation properties in this fragment will move to the `ConversationSnippet` once I finish implementing the connection on the metaphysics side